app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/OfflineDatabaseService.kt

```diff
@@ -258,4 +258,12 @@ class OfflineDatabaseService internal constructor(
         val user = userDao.load(uid, type) ?: return null
         return fromUser(user)
     }
+
+    override suspend fun getStatistic(user: User.Id, dataset: Id): Statistic? {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun putStatistic(statistic: Statistic) {
+        TODO("Not yet implemented")
+    }
```

Comment:
Is this going to be implemented in future PR's ? Maybe it is because Jonas is working on this part ?

Reply:
We should maybe add an issue, just to remember to address this